### PR TITLE
[INFRA-317] rewrite to use Ansible

### PR DESCRIPTION
**Pull request by kotchaosu**
This is nice

```
quay.io/syncano/python-codebox   test                78948a97d6e6        2 hours ago         1.266 GB
quay.io/syncano/python-codebox   latest              dc1d6a90bcca        6 days ago          1.464 GB
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,66 +1,9 @@
...
-    ln -sf /home/syncano/p3v5.0/bin/python /usr/bin/python3 && \
-    ln -sf /home/syncano/p3v5.0/bin/python /usr/bin/python3-lib5.0
-
-# -- CUT END --
-USER syncano
-WORKDIR /tmp
-CMD "python"
+RUN apt-get update && apt-get install -y openssh-server
+RUN mkdir /var/run/sshd /root/.ssh
+COPY python/files/id_rsa.pub /root/.ssh/authorized_keys
```
Comment (23doors) [inline Dockerfile]: it's just some random ssh key used for installation right? Maybe we should remove it and disable ssh after we're done. Seems like a potential security threat at some point

**Review by 23doors — COMMENTED** [review]
looks good. just maybe that public ssh could be made a little more secure, just in case

File: Dockerfile (hunk 1 above)
Comment (kotchaosu) [reply Dockerfile]: ok

Comment (kotchaosu) [reply Dockerfile]: we remove it during deploy - I'll leave it here for sake of simplicity

**Pull request merged**